src/dotnet/commands/dotnet-compile/Program.cs

```diff
@@ -71,16 +71,35 @@ private static bool OnExecute(List<ProjectContext> contexts, CompilerCommandApp
 
             var managedOutput = outputPaths.CompilationFiles.Assembly;
 
+            // Create the library exporter
+            var exporter = context.CreateExporter(args.ConfigValue);
```

Comment:
How does this even work? Don't you need to pass in the runtime assemblies to the AOT compiler? This will get you the wrong set of assemblies. You need the project context for the AOT runtime ID, which isn't in the projects by default. Tons of work to do here.